### DISCUSSION TITLE
End-to-End encryption (E2EE) support

### DISCUSSION THREAD

**Discussion by jibon57** [8xFF/atm0s-media-server#366] [Ideas]
As most of the browser E2EE now so having this feature will be nice. Ref: https://github.com/livekit/client-sdk-js/tree/main/src/e2ee

**Comment by giangndm** [comment]
@jibon57 I found a simple example here: https://webrtc.github.io/samples/src/content/insertable-streams/endtoend-encryption/

May be we can start from simplest API then take a lock with advance implement from livekit
